src/library_webgl2.js

```diff
@@ -730,8 +730,11 @@ var LibraryWebGL2 = {
       return;
     }
     var ret = GLctx.getSyncParameter(GL.syncs[sync], pname);
-    {{{ makeSetValue('length', '0', 'ret', 'i32') }}};
-    if (ret !== null && length) {{{ makeSetValue('length', '0', '1', 'i32') }}}; // Report a single value outputted.
+    var notNull = ret !== null;
+    if (notNull) {
```

Comment:
Oh actually, can you fuse these two statements to one? i.e. `if (ret !== null)` since `notNull` is now only referenced once. Closure can be dumb about optimizing this kind of stuff.

Reply:
Done!